experimental/add2app/example_module/README.md

```diff
@@ -0,0 +1,3 @@
+# example_module
+
+A simple module used to demo Flutter's Add2App features
```

Comment:
A simple module to be embedded in other host apps, used to demo Flutter's add-to-app process.

Reply:
Done.